test/jdk/java/nio/channels/Selector/SelectWithConsumer.java

```diff
@@ -762,4 +757,32 @@ static ByteBuffer messageBuffer() {
             throw new RuntimeException(e);
         }
     }
+
+    /**
+     * Returns the current time in milliseconds.
+     */
+    private static long millisTime() {
+        long now = System.nanoTime();
+        return TimeUnit.MILLISECONDS.convert(now, TimeUnit.NANOSECONDS);
```

Comment:
you can get rid of local variable now as follows.
return TimeUnit.MILLISECONDS.convert(System.nanoTime(), TimeUnit.NANOSECONDS);

Reply:
> [...] you can get rid of local variable now as follows.

I think I'll forego this change for now.